lms/services/exceptions.py

```diff
@@ -12,10 +12,6 @@ class LTILaunchVerificationError(ServiceError):
     """
 
 
-class NoConsumerKey(LTILaunchVerificationError):
-    """Raised when a launch request has no ``oauth_consumer_key`` parameter."""
```

Comment:
This isn't something we retrieve any more, `oauthlib` does it for us. This means we never have an opportunity to raise this.